rules/thrift_2.mk

```diff
@@ -0,0 +1,20 @@
+# thrift package
+
+THRIFT_2_VERSION = 0.14.1
```

Comment:
THRIFT_2, i am not sure why naming it as thrift_2 but not 14?

Reply:
I don't want to make variables and files/folders version dependent as then wi will need to rename them each time when bump version. As src/thrift and THRIFT_VERSION is currently used by the old thrift 0.11.0 target I named this target `thrift_2`. Once this and https://github.com/opencomputeproject/SAI/pull/1403 will be merged, and SONiC will move to the new SAI refpoint we could mark thrift 0.11.0 as deprecated and when all vendors will be ready remove thrift 0.11.0. Then it will be possible to remove this `_2' suffix from names/pathes

Reply:
Great, and thanks for sharing the plan.

Reply:
Even you mentioned "don't want to make variables and files/folders version dependent", the `_2` acts the same as `_0_14_1`. And since you have a roadmap to rename it anyway in future, currently it still make sense to use `_0_14_1`.

Reply:
Done. But for some reason CI for Mellanox failed with
```
Connecting to www.mellanox.com (www.mellanox.com)|151.101.54.228|:443... connected.
ERROR: The certificate of 'www.mellanox.com' is not trusted.
ERROR: The certificate of 'www.mellanox.com' has expired.
The certificate has expired
make[1]: *** [Makefile:34: /sonic/target/debs/buster/mft_4.18.1-16_amd64.deb] Error 5
make[1]: Leaving directory '/sonic/platform/mellanox/mft'
[  FAIL LOG END  ] [ target/debs/buster/mft_4.18.1-16_amd64.deb ]
make: *** [slave.mk:486: target/debs/buster/mft_4.18.1-16_amd64.deb] Error 1
```

Reply:
Seem a temporary issue on the server side.